libraries/standard/mqtt/integration-test/mqtt_system_test.c

```diff
@@ -311,18 +312,19 @@ static void establishMqttSession( MQTTContext_t * pContext,
     networkBuffer.pBuffer = buffer;
     networkBuffer.size = NETWORK_BUFFER_SIZE;
 
-    /* Application callbacks for receiving incoming publishes and incoming acks
+    /* User-callback for receiving incoming publishes and incoming acks
      * from MQTT library. */
-    callbacks.appCallback = eventCallback;
+    userCallback = eventCallback;
 
-    /* Application callback for getting the time for MQTT library. This time
+    /* Platform-specific utility function for getting the time for MQTT library. This time
      * function will be used to calculate intervals in MQTT library.*/
-    callbacks.getTime = Clock_GetTimeMs;
+    getTimeFunction = Clock_GetTimeMs;
```

Comment:
In demos, these additional steps are providing more readability for the example application. Can we avoid it here and directly pass the function pointer in MQTT_Init?

Reply:
Removed it